modules/library/main/src/test/java/org/geotools/geometry/jts/GeometryClipperTest.java

```diff
@@ -1,3 +1,20 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ * 
+ *    (C) 2015, Open Source Geospatial Foundation (OSGeo)
```

Comment:
Thanks for adding the copyright headers. The year should be 2010 - 2015, as shown by the history of file at:

````
svn log http://svn.osgeo.org/geotools/trunk/modules/library/main/src/test/java/org/geotools/geometry/jts/GeometryClipperTest.java
````